openquake/server/templates/engine/index.html

```diff
@@ -60,6 +60,25 @@
               <a href="#" data-calc-id="<%= calc.get('id') %>" is-running="<%= calc.get('is_running') %>" class="btn btn-sm btn-log" style="margin: 2px 0 2px 0;">Console</a>
               <% if (calc.get('is_running') == false) { %>
                 <a href="#" data-calc-id="<%= calc.get('id') %>" class="btn btn-sm btn-danger" style="margin: 2px 0 2px 0;">Remove</a>
+                <!-- confirm_modal -->
+                <div class="modal-backdrop  in hide back_confirmDialog<%= calc.get('id') %>"></div>
```

Comment:
classes name should be spaced by one space (even if having two spaces does not break things) since we are touch the code let's clean it where it's possible.